packages/rs-drive-abci/.env.local

```diff
@@ -85,3 +85,6 @@ BLOCK_SPACING_MS=5000
 TOKIO_CONSOLE_ENABLED=false
 TOKIO_CONSOLE_ADDRESS=127.0.0.1:6669
 TOKIO_CONSOLE_RETENTION_SECS=180
+
+GROVEDB_VISUALIZER_ENABLED=true
```

Comment:
Disable by default

Reply:
I think for local it should be enabled @shumkov

Reply:
I believe 95% of cases you don't need it when you run local. If you need you just enable it.